flixel/input/keyboard/FlxKey.hx

```diff
@@ -79,7 +81,10 @@ abstract FlxKey(Int) from Int to Int
 	var LEFT = 37;
 	var RIGHT = 39;
 	var TAB = 9;
+	var START = 15;
```

Comment:
This should just be called WINDOWS, I can't find any other places referring to this as the  "start" key